packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -27,7 +27,7 @@ var get = Ember.get;
   @class BuildURLMixin
   @namespace DS
 */
-var BuildURLMixin = Ember.Mixin.create({
+export default Ember.Mixin.create({
```

Comment:
@rwjblue you may have identified a good refactoring, although unrelated to this point.

default export should really have some extremely strong meaning to the file it represents. As that is it's public addressable location. 

Property exports do not need this, that is also why property exports are named. ` export function foo() { }`